tests/src/PatternAssertion/GalleryPatternAssert.php

```diff
@@ -83,6 +83,7 @@ protected function assertItems(array $expected, Crawler $crawler): void {
         'image' => $item['media']['rendered'],
         'caption_title' => $item['media']['caption_title'] ?? NULL,
         'caption' => $item['media']['caption_media'] ?? NULL,
+        'is_playable' => $item['media']['is_playable'] ?? NULL,
```

Comment:
Seems you wanted to use this, I think your idea was to set this send to the carousel expected items and asset there that it matches? anyway, in the end you did not set any actual values to it, nor are you using it in the carousel pattern.

Reply:
Indeed, i removed it because it's already tested inside the carousel test directly using $isPlayable.